lib/gateway/nginx.conf.tpl

```diff
@@ -40,7 +40,7 @@ http {
         add_header X-XSS-Protection "1; mode=block";
         add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
         # --
-        
+
```

Comment:
This file only seems to have whitespace changes, I am not sure if we need to commit this.

Reply:
Indeed. The commit name is "whitespace" and it only does that.
The rationale is that it will help detecting "real" changes in future commits.